cypress/integration/users/delete-users.spec.js

```diff
@@ -67,6 +68,6 @@ describe('Deleting user', () => {
     ]);
 
     cy.visit(`/users/preview/${id}`);
-    cy.expect(ResultsPane.has({ id: 'pane-userdetails' }));
+    cy.expect(KeyValue().exists({ value: lastName }));
```

Comment:
`exists()` does not take any arguments. Instead it checks that an element matching your locator and filters exists.

 In the case of `KeyValue()` without arguments, there are no locator or filters, and `exists()` takes no arguments, so as-is, this will only assert that there is _some_ `KeyValue` present. Any `KeyValue` will satisfy it. 

Instead, I would locate a specific key value and then use `has()` to make an assertion on its expected value: E.g.

```js
    cy.expect(KeyValue('Last Name').has({ value: lastName }));
```

Implicit in this assertion is the assertion that `KeyValue('Last Name').exists()`

Reply:
(I don't know what the key is for your `KeyValue`, `'Last Name'` was just a guess

Reply:
fixed it

Reply:
@poflankov you actually found a bug in the docs! Thanks!

Reply:
Fix is here https://github.com/thefrontside/interactors/pull/84